SU2_CFD/include/solvers/CFVMFlowSolverBase.inl

```diff
@@ -979,10 +979,8 @@ void CFVMFlowSolverBase<V, R>::LoadRestart_impl(CGeometry **geometry, CSolver **
   {
   /*--- Delete the class memory that is used to load the restart. ---*/
 
-    delete [] Restart_Vars;
-    Restart_Vars = nullptr;
-    delete [] Restart_Data;
-    Restart_Data = nullptr;
+    Restart_Vars.clear(); Restart_Vars.shrink_to_fit();
+    Restart_Data.clear(); Restart_Data.shrink_to_fit();
```

Comment:
```suggestion
    Restart_Vars = decltype(Restart_Vars){};
    Restart_Data = decltype(Restart_Data){};
```

Reply:
That's better, as `shrink_to_fit` tends to free but does not guarantee it. Thanks :+1: